src/memory-analyzer/gdb_api.cpp

```diff
@@ -282,7 +283,7 @@ bool gdb_apit::run_gdb_to_breakpoint(const std::string &breakpoint)
   command += " " + breakpoint;
 
   write_to_gdb(command);
-  if(!most_recent_line_has_tag("^done"))
+  if(!check_command_accepted())
```

Comment:
This would be unreachable as `check_command_accepted()` has a `CHECK_RETURN()` that already checks for that. We probably have to move the `CHECK_RETURN` to the call site of `check_command_accepted()`.

Reply:
We could do that. What do you think about the solution in the next commit?

Reply:
I think we can remove the bool return value from `check_command_accepted()`. Otherwise looks good!

Reply:
You're right of course, fixed.